api/accountsGET.ts

```diff
@@ -0,0 +1,10 @@
+import handler from './handler';
+import db from './db';
+import { QueryResult } from 'pg';
+
+export default handler(async (request: any) => {
+  return await db(async (connection) => {
+    let results: QueryResult = await connection.query("SELECT authorityid, username, authorityname, authoritytype FROM authority");
+    return results.rows;
+  });
+});
```

Comment:
Add authentication to this endpoint.
```suggestion
}, auth.authenticate);
```

You'll need to import the authenticate module to do so:
`import * as auth from './authenticate';`